crates/bevy_window/src/window.rs

```diff
@@ -54,6 +73,7 @@ pub struct Window {
     requested_height: f32,
     physical_width: u32,
     physical_height: u32,
+    resize_constraints: WindowResizeConstraints,
```

Comment:
There is no command to change this.

Reply:
Should I remove this or create a command?

Reply:
You should probably add a command to change it